docs/supervisor-deploy-with-image-from-private-registry.md

```diff
@@ -26,7 +26,7 @@ Manifest
 
 *   Velero vSphere operator YAML: [https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml)
 
-Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version,  please use this [YAML file](https://vmwaresaas.jfrog.io/ui/native/Velero-YAML/Velero/SupervisorService/1.4.0/velero-vsphere-1.4.0-def.yaml).
+Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version,  please find latest version in this [page](https://github.com/vsphere-tmm/Supervisor-Services/blob/main/README.md#velero-versions).
```

Comment:
nit: "in this page" -> "on this page"
nit: double spaces before "please". 
nit: Remove "please".

Reply:
Fixed.